src/main/java/com/google/api/codegen/metacode/InitCodeGenerator.java

```diff
@@ -237,6 +237,11 @@ private InitCodeLine generateCodeInit(
         }
         initValueConfig = initValueConfig.withInitialValue(validatedValue);
       }
+
+      // This is used to make type aliases for samples.
+      if (typeRef.isMessage() && context.typeTable() != null) {
```

Comment:
I was following php and made enums set to nil. I realized my mistake. The typeref.ismessage() will be removed in my next commit.